presto-docs/src/main/sphinx/release/release-0.293.rst

```diff
@@ -0,0 +1,149 @@
+* Fix a bug where a mirrored :func:`arrays_overlap(x, y) -> boolean` function does not return the correct value. `#23845 <https://github.com/prestodb/presto/pull/23845>`_
+* Fix returning incorrect results from the "second" UDF when a timestamp is in a time zone with an offset that is at the granularity of seconds. `#25090 <https://github.com/prestodb/presto/pull/25090>`_
+* Fix the issue of sensitive data such as passwords and access keys being exposed in logs by redacting sensitive field values. `#24886 <https://github.com/prestodb/presto/pull/24886>`_
+* Fix RedisProviderPlugin loading. `#24835 <https://github.com/prestodb/presto/pull/24835>`_
+* Improve memory usage in reader with nested readers by resetting all nested readers. `#24912 <https://github.com/prestodb/presto/pull/24912>`_
+* Improve ACL check by moving checkQueryIntegrity from Dispatch phase to Analyzer phase. `#24927 <https://github.com/prestodb/presto/pull/24927>`_
+* Improve communication between coordinator and worker with thrift serde. `#25079 <https://github.com/prestodb/presto/pull/25079>`_
+* Improve communication between coordinator and worker with thrift serde. `#25020 <https://github.com/prestodb/presto/pull/25020>`_
+* Improve how we merge multiple operator stats together. `#24921 <https://github.com/prestodb/presto/pull/24921>`_
+* Improve metrics creation by refactoring local variables to a dedicated class. `#24921 <https://github.com/prestodb/presto/pull/24921>`_
+* Improve performance of ``ORDER BY`` queries on single node execution. `#25022 <https://github.com/prestodb/presto/pull/25022>`_
+* Improve query plans using the ``SimplifyPlanWithEmptyInput`` optimizer to convert a table scan which returns no data to an empty values node. `#25155 <https://github.com/prestodb/presto/pull/25155>`_
+* Improve performance of ``LOJ + IS NULL`` queries by adding distinct on right side of semi-join for it. `#24884 <https://github.com/prestodb/presto/pull/24884>`_
+* Add DDL SQL support for ``SHOW CREATE SCHEMA``. `#24356 <https://github.com/prestodb/presto/pull/24356>`_
+* Add authentication capabilities to Presto router. `#24407 <https://github.com/prestodb/presto/pull/24407>`_
+* Add configuration property ``hive.metastore.catalog.name`` to pass catalog names to the metastore, enabling catalog-based schema management and filtering. `#24235 <https://github.com/prestodb/presto/pull/24235>`_
+* Add coordinator health checks to Presto router. `#24449 <https://github.com/prestodb/presto/pull/24449>`_
+* Add counter JMX metrics to Presto router. `#24449 <https://github.com/prestodb/presto/pull/24449>`_
+* Add example custom scheduler plugin - Metrics based custom scheduler plugin. `#24439 <https://github.com/prestodb/presto/pull/24439>`_
+* Add support for custom scheduler plugin. `#24439 <https://github.com/prestodb/presto/pull/24439>`_
```

Comment:
```suggestion
* Add support for custom scheduler plugin to the Presto router. `#24439 <https://github.com/prestodb/presto/pull/24439>`_
```

Put in dedicated section for Router.  Comb through and find other things router related and put there as well.